Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -85,12 +88,26 @@ extension AdjacencyList: MutableGraph {
 		return EdgeId(source: source, destination: destination, offset: offset)
 	}
 
-	public mutating func removeEdge(from u: VertexId, to v: VertexId) {
+    /// Removes the edge (u, v) from the graph.
```

Comment:
For consistency with addEdge and completeness
```suggestion
    /// Removes all edges from `u` to `v`.
```